scalding-core/src/main/scala/com/twitter/scalding/bdd/TBddDsl.scala

```diff
@@ -68,7 +68,7 @@ trait TBddDsl extends FieldConversions with TypedPipeOperationsConversions {
 
       val outputPipe = operation(inputPipes) map { Tuple1(_) }
 
-      outputPipe.write(TypedTsv[Tuple1[OutputType]]("output"))
+      outputPipe.write(FixedPathTypedDelimited[Tuple1[OutputType]]("output", "\t"))
```

Comment:
This change is made just because this code is in scalding core so it can't use the macros